gradle.properties

```diff
@@ -3,5 +3,6 @@ pluginsUrl=
 uploaderDestination=
 uploaderCredentialsFile=
 uploaderMultithreadedUpload=
+verboseTestOutput=
```

Comment:
Should we make this explicitly false?

Reply:
That actually doesn't work, I think gradle is just treating the property as a string and will treat it as false in a conditional only if it's empty.